tests/taskcluster/tasks/raptor_decision.yml

```diff
@@ -0,0 +1,35 @@
+reruns: 4
+task:
+  metadata:
+    name: 'Gaia Raptor Decision Task'
+    description: 'Raptor performance tests'
+
+  workerType: '{{workerType}}'
+  provisionerId: '{{provisionerId}}'
+  scopes:
+    # Source caches contains various sources (including a tarball of gaia source
+    # tree)
+    - docker-worker:cache:tc-vcs
+    - docker-worker:image:quay.io/mozilla/gaia-taskenv:*
+
+  payload:
+    maxRunTime:
+    image: quay.io/mozilla/gaia-taskenv:0.8.9
+    cache:
+      tc-vcs: /home/worker/.tc-vcs
+    command:
+      - entrypoint
+      - npm install taskcluster-client promise && echo 'what' && node tests/taskcluster/bin/raptor_graph > /raptor_graph.json && cat /raptor_graph.json
```

Comment:
Oh looks like that's a carry-over from:
https://github.com/mozilla-b2g/gaia/blob/master/taskgraph.json#L33
Don't know why it's there. I'll remove it from my task, thanks :)